src/azure.js

```diff
@@ -82,3 +82,86 @@ api.declare({
     expiry:   expiry.toJSON()
   });
 });
+
+api.declare({
+  method:     'get',
+  route:      '/azure/:account/blob/:container/:level',
```

Comment:
Also we should define a pattern for the `container` parameter here:
https://github.com/elenasolomon/taskcluster-auth/blob/d1eb601a2b9b5f2709f4c0f672a7d060d99169d1/src/v1.js#L57

Perhaps we can even define `level` there too... Since level is always `/^(read-write|read-only)$/`